Tests/GitHubCodespaces.tests.ps1

```diff
@@ -34,6 +34,59 @@ BeforeAll {
     $mainBranchName = $repo | Get-GitHubRepositoryBranch | Select-Object -ExpandProperty name -First 1
 }
 
+Describe 'GitHubCodespaces\Set-GitHubCodespaceVisibility' {
+    Context 'When setting the visibility of a codespace' {
+
+        It 'sets the visibility successfully too ''disabled'' ' {
+            $setVisibilityParams = @{
+                Force = $true
+                OrganizationName = $script:organizationName
+                Visibility = 'disabled'
+            }
+            Set-GitHubCodespaceVisibility @setVisibilityParams
+        }
+
+        It 'sets the visibility successfully too ''all_members'' ' {
+            $setVisibilityParams = @{
+                Force = $true
+                OrganizationName = $script:organizationName
+                Visibility = 'all_members'
+            }
+            Set-GitHubCodespaceVisibility @setVisibilityParams
+        }
+        It 'sets the visibility successfully too ''selected_members'' ' {
+            $setVisibilityParams = @{
+                Force = $true
+                OrganizationName = $script:organizationName
+                UserName = $script:ownerName
+                Visibility = 'selected_members'
+            }
+            Set-GitHubCodespaceVisibility @setVisibilityParams
+        }
+
+    }
+}
+
+Describe 'GitHubCodespaces\Remove-GitHubCodespaceUser' {
```

Comment:
Can we run this test successfully multiple times by itself, or does it depend on a previous test to set up the correct state?  In other words, if I remove `octocat` the first time, it shouldn't throw.  If I then try to remove `octocat` a second time, does the API care, or will I get a different response?